Dev3/Dev3/Company.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Dev3
+{
+    /// <summary>
+    /// Class containing information about employees.
+    /// </summary>
+    class Company
+    {
+        const int juniorCount = 40; //The number of total Juniors in the company.
+        const int middleCount = 30; //The number of total Middles in the company.
+        const int seniorCount = 10; //The number of total Seniors in the company.
+        const int teamleadCount = 5; //The number of total Team Leads in the company.
+
+        private List<Employee> ListOfEmployees { get; set; } = new List<Employee>();
+
+        public Company()
+        {
+            for (int i = 0; i < teamleadCount; i++)
```

Comment:
looks a little weird. Is it better to implement some methods like 'AddEmployee' or 'HireEmployee' ?